### PR TITLE
Clean Up unnecessary UI

### LINKED ISSUE #18: Clean-Up unnecessary UI features

### PR DESCRIPTION
**Pull request by sipian**
Closes #18 

Authored-by: sipian <harshaga97@gmail.com>

### CODE REVIEW

File: src/components/layout/MainFooter.js [hunk 1]
```diff
@@ -44,23 +44,7 @@ MainFooter.defaultProps = {
...
-      to: "#"
-    },
-    {
-      title: "Products",
-      to: "#"
-    },
-    {
-      title: "Blog",
-      to: "#"
+      to: "/home"
```
Comment (b-z) [inline src/components/layout/MainFooter.js:47]: do we have this `/home`?

File: src/components/layout/MainNavbar/MainNavbar.js [hunk 2]
```diff
@@ -4,42 +4,46 @@ import classNames from "classnames";
...
+        <
+        Container className = "p-0" >
+        <
+        Navbar type = "light"
+        className = "align-items-stretch flex-md-nowrap p-0" >
+        <
+        NavbarSearch / > { /* <NavbarNav /> */ } <
+        NavbarToggle / >
+        <
+        /Navbar> < /
```
Comment (b-z) [inline src/components/layout/MainNavbar/MainNavbar.js]: do not use beautifier here, i think it is not standard js file.

File: src/components/layout/MainSidebar/SidebarMainNavbar.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ class SidebarMainNavbar extends React.Component {
         >
           <NavbarBrand
             className="w-100 mr-0"
-            href="#"
+            href="/home"
             style={{ lineHeight: "25px" }}
           >
             <div className="d-table m-auto">
```
Comment (b-z) [inline src/components/layout/MainSidebar/SidebarMainNavbar.js:33]: @VamshiTeja has said that this file seems not used

File: src/components/layout/MainFooter.js (hunk 1 above)
Comment (sipian) [reply src/components/layout/MainFooter.js:47]: yes we have this

**Pull request merged**